networkx/drawing/tests/test_layout.py

```diff
@@ -402,7 +411,7 @@ def test_rescale_layout_dict(self):
         for k, v in expectation.items():
             assert (s_vpos[k] == v).all()
         s_vpos = nx.rescale_layout_dict(vpos, scale=2)
-
+        assert s_vpos == {0: (-2, -2), 1: (2, 2), 2: (0, 0)}
```

Comment:
I think this line is causing the test failures. Does it fail locally?

Reply:
@dschult I rebased the code. I think I pulled the wrong main which was not up-to-date. Should be ok now.